README.md

```diff
@@ -168,6 +168,15 @@ EventsApi api = new ApiClient("API KEY", "API-SECRET")
                 .createService(EventsApi.class);
 ```
 
+By default, the Java SDK will upload to the [US1 Data Center URL](https://docs.mparticle.com/developers/server/http/#endpoint).  If the API key you're sending to exists in an mParticle Data Center that's not US1, you can add the following configuration to the SDK initialization:
+
+```java
+Retrofit.Builder mybuilder = new Retrofit.Builder().baseUrl("DATA CENTER URL").build();
+EventsApi api = new ApiClient("API KEY", "API-SECRET")
+                .setAdapterBuilder(mybuilder)
+                .createService(EventsApi.class);
+```
+
```

Comment:
[See the mParticle docs site](https://docs.mparticle.com/developers/data-localization/#events-api) to find the right URL for your hosting location.

Reply:
@samdozor I added the piece about defaulting to US1 Data Center, which links to mP docs.  What are you suggesting with this comment?